docs/config.md

```diff
@@ -47,7 +47,7 @@ name = "2012_Ford_Focus"
 type = "single_fuel"
 # the file for the routee-powertrain model
 model_input_file = "models/2012_Ford_Focus.bin"
-# what underlying machine learn framework to use [smartcore | onnx]
```

Comment:
noting this comment, maybe it should still list onnx as a possibility

Reply:
yeah good point, I'll add that back in